core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/InstrumentationDirectives.scala

```diff
@@ -57,11 +59,15 @@ trait InstrumentationDirectives extends RawlsInstrumented with TracingDirectives
     ))
 
   private val redactWorkspaceNames =
-    (Slash ~ "api").? / "workspaces" / (!"entities" ~ Segment) / (Segment ~ SegmentIgnore.repeat(0,
-                                                                                                 Int.MaxValue,
-                                                                                                 separator = Slash
+    (Slash ~ "api").? / "workspaces" / (!("entities" | "id") ~ Segment) / (Segment ~ SegmentIgnore.repeat(0,
```

Comment:
this and the following change made me look for better and more future proof ways to figure out a standard name for an api "route".  We have this all in swagger, why not parse that?